ee/server/service/orbit.go

```diff
@@ -136,6 +136,11 @@ func (svc *Service) GetOrbitSetupExperienceStatus(ctx context.Context, orbitNode
 
 	}
 
+	_, err = svc.SetupExperienceNextStep(ctx, host.UUID)
```

Comment:
This is to force progress of next step every time this endpoint gets polled? I assume it is idempotent/no harm calling even if it should stay on current step?

Reply:
If there's nothing to do, it should change nothing, although this is probably good to make sure we enqueue the first step

Reply:
yep! noop if it shouldn't/can't proceed to next step.